src/Measurements.jl

```diff
@@ -128,6 +121,19 @@ include("math.jl")
 include("linear_algebra.jl")
 include("show.jl")
 include("parsing.jl")
-include("plot-recipes.jl")
+
+if !isdefined(Base,:get_extension)
+    using Requires
+    using RecipesBase
+    include("../ext/MeasurementsRecipesBaseExt.jl")
```

Comment:
Is it intentional that only this single extension is outside of the next block?

Reply:
this package supports julia 1.0. for some reason, using one `isdefined` block failed.

Reply:
I see, thanks.